src/d3d9/d3d9_device.cpp

```diff
@@ -1880,18 +1880,30 @@ namespace dxvk {
           break;
 
         case D3DRS_COLORWRITEENABLE:
+          if (likely(m_state.renderTargets[0] != nullptr))
```

Comment:
This is per-channel, we should only dirty it if ``!!old != !!new``

I guess same goes for UpdateActiveRTs